modules/rosa-sts-oidc-provider-command.adoc

```diff
@@ -21,8 +21,9 @@ When using `manual` mode, the `aws` command is printed to the terminal for your
 [source,terminal]
 ----
 aws iam create-open-id-connect-provider \
-	--url https://rh-oidc.s3.<aws_region>.amazonaws.com/<cluster_id> \
+	--url https://rh-oidc.s3.<aws_region>.amazonaws.com/<cluster_id> \ <1>
```

Comment:
From v1.2.7 the output ROSA generates for the manual mode of `rosa create oidc provider` is going to be something like this:
`--url https://d3gt1gce2zmg3d.cloudfront.net/20eh31t5vaasiidbumcjfbnknb6c6883`